### PR TITLE
[Enhancement] Add detailed log information when an error occurs during the MV refresh operation about list partion basetable

### PR DESCRIPTION
**Pull request by zhangheihei**
## Why I'm doing:
Add detailed log information when an error occurs during the MV refresh operation about list partion basetable
image

## What I'm doing:
add detail log

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Comment by zhangheihei** [comment]
hi @LiShuMing  plz review the code

**Review by LiShuMing — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java [hunk 1]
```diff
@@ -1045,7 +1045,8 @@ public void sendDropAutoIncrementMapTask() {
      * @return : table's partition name to range partition key mapping.
      */
     public Map<String, Range<PartitionKey>> getRangePartitionMap() {
-        Preconditions.checkState(partitionInfo instanceof RangePartitionInfo);
+        Preconditions.checkState(partitionInfo instanceof RangePartitionInfo, "Materialized view related base table:%s " +
```
Comment (trueeyu) [inline fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java:1048]: It is best to report errors at the upper level. Reporting MV errors here can easily cause misunderstandings.

Comment (zhangheihei) [reply fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java]: tks. Optimize error messages

**Review by LiShuMing — APPROVED** [review]
LGTM

**Pull request closed without merging**